app/models/handler.rb

```diff
@@ -23,6 +23,9 @@ def service
 
   def handle(payload)
     service.new(settings, payload).call
+
+    self.handle_count += 1
```

Comment:
This is basically `increment!(:handle_count)`. Although do we want to find a threadsafe way to do this too?

Reply:
https://apidock.com/rails/ActiveRecord/CounterCache/ClassMethods/increment_counter

Reply:
increment_counter TIL.

> Although do we want to find a threadsafe way to do this too?

I'd say no for now. If we had a job infrastructure yet probably, but I'd rather missing few count increment than risking a deadlock for now?

It was more of a "we have nothing, let's have something even if it is more or less accurate"

Reply:
Looks like `increment_counter` is threadsafe (not transaction save, which we do not need here)